lib/tasks/grda_warehouse.rake

```diff
@@ -288,7 +296,8 @@ namespace :grda_warehouse do
 
   desc 'Monthly tasks'
   task monthly: [:environment, 'log:info_to_stdout'] do
-    GrdaWarehouse::Tasks::CensusImport.new.run!
+    # this runs every day out of RunDailyImportsJob
+    # GrdaWarehouse::Tasks::CensusImport.new.run!
```

Comment:
doesn't seem like we need this here?

Reply:
Hmm, I think at some point, this monthly one was a full-rebuild.  Maybe we don't need that anymore, or maybe we should update this so that it does a full-rebuild.  Thoughts?

Reply:
discussed 6/13 and decided to just remove the monthly call as it processes 3yrs daily